<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QDialog" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <widget class="QPushButton" name="main">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit To Main</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitInfo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
